### PR TITLE
Replace jquery event with angular event

### PR DESCRIPTION
**Pull request by gauravshe**
No need of run scope.$apply every time, Only run when an update is called from a non angular event.
Signed-off-by: Gaurav Shekhadiya <gauravshe@turn.com>

### CODE REVIEW

**Comment by bcherny** [comment]
Instead of this hack, can you remove the manually attached listener on [keyUp and paste](https://github.com/gauravshe/angular-search-1/blob/8e19e4bc03b3a7e8b3b5b01cb588194bb3b6e84b/dist/search.js#L91) and call `scope.update` using ngKeyUp and ngPaste in the template instead? That would avoid the need for manually calling scope.$apply.

**Comment by gauravshe** [comment]
Removed jquery event and replaced with angular event

File: src/search.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	<input
 		type="search"
 		ng-model="param"
+		ng-keyup="change($event)"
```
Comment (bcherny) [inline src/search.html:7]: What about the paste event? Consider using a combination of ngChange and ngPaste.

File: bower.json [hunk 2]
```diff
@@ -11,11 +11,11 @@
   ],
   "license": "Apache2",
   "dependencies": {
-    "angular": "1.0 - 1.1",
+    "angular": "1.2.0",
```
Comment (bcherny) [inline bower.json]: You probably want `^1.2.0`

File: bower.json [hunk 3]
```diff
@@ -11,11 +11,11 @@
...
     "jquery": "*"
   },
   "devDependencies": {
-    "angular-mocks": "1.0 - 1.1",
+    "angular-mocks": "1.2.0",
```
Comment (bcherny) [inline bower.json]: Same here

File: src/search.html (hunk 1 above)
Comment (gauravshe) [reply src/search.html:7]: added ng-paste event

**Comment by gauravshe** [comment]
bcherny can you please review and merge this PR as this ticket is pending since long.

**Comment by gauravshe** [comment]
@bcherny can you please look this PR.

**Comment by bcherny** [comment]
Published 0.2.1.

**Pull request merged**